src/pages/steps/AdditionalInformationStep/PriceInformation.tsx

```diff
@@ -247,7 +247,7 @@ const PriceInformation = ({
             errors.rates?.at(index)?.type;
 
           return (
-            <Stack key={`rate_${rate.id}`}>
+            <Stack key={`rate_${rate.name}_${rate.price}_${rate.category}`}>
```

Comment:
The `id` comes from the `fields` object originally that's why I had to build the `controlledFields` like in their example to get the benefit from it https://react-hook-form.com/api/usefieldarray/#shouldUnregister-example

Reply:
Ok, good to know 👍🏻